tests/unit/s2n_ktls_test_utils_test.c

```diff
@@ -21,6 +21,7 @@
 
 #define S2N_TEST_TO_SEND     10
 #define S2N_CONTROL_BUF_SIZE 100
+#define S2N_TEST_MSG_IOVLEN  5
```

Comment:
Does `IOVLEN` have special meaning? Or would an underscore make sense?
```suggestion
#define S2N_TEST_MSG_IOV_LEN  5
```

Reply:
`msg_iovlen` is a thing on `msghdr` https://man7.org/linux/man-pages/man2/sendmsg.2.html